extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -23,6 +25,7 @@ const buildFiles: string[] = [
 	'System.ComponentModel.Composition.dll',
 	'Microsoft.Data.Tools.Schema.SqlTasks.targets'
 ];
+const existsAsync = promisify(fs.exists);
```

Comment:
can utils.exists() be used instead?

Reply:
utils do access in place of exist? any reason?

Reply:
I think fs.exists was getting deprecated and maybe also because access returns a promise. @Benjin probably knows better since he suggested adding this method to utils

Reply:
https://stackoverflow.com/questions/28532820/fs-exists-fs-existssync-why-are-they-deprecated

You can promisify existsSync if you want, but generally it's preferable to just stick to using the utils.exists since it's already there.

Reply:
also, I think stat is used for existence checking now